core/src/avm2/globals/flash/desktop/IFilePromise.as

```diff
@@ -0,0 +1,24 @@
+package flash.desktop {
+    import flash.utils.IDataInput;
+    import flash.events.ErrorEvent;
+
+    [API("668")] // AIR 2.0
+    public interface IFilePromise {
+
+        [API("668")]
```

Comment:
Aren't these unnecessary since the interface itself is version-gated?

Reply:
Probably, but it doesn't hurt to be sure

Reply:
Personally I think we should avoid adding what is basically noise. If we have bugs with methods/properties being exposed unexpectedly those can be fixed.